docs/core/diagnostics/faq-dumps.yml

```diff
@@ -11,15 +11,48 @@ sections:
+
+          1. Either the calling process needs to have the same user id as the target process, or the calling process needs to have CAP_SYS_PTRACE. If neither of these is true access is denied. Since the .NET runtime doesn't do anything to change the user account when launching createdump the user ids should match and this step should succeed.
+          2. If createdump doesn't have CAP_SYS_PTRACE (it doesn't by default), then the target process being dumped needs to be marked "dumpable". By default most processes on Linux are dumpable but this setting can be changed by calling [prctl()](https://www.man7.org/linux/man-pages/man2/prctl.2.html) with the PR_SET_DUMPABLE option. If you add capabilities to a process using the setcap tool, this can also cause a process to stop being dumpable. See [the Linux documentation](https://www.man7.org/linux/man-pages/man2/PR_SET_DUMPABLE.2const.html) for a more detailed description of the dumpable setting and what causes it to be disabled.
+          3. All of the enabled [Linux security modules](https://www.kernel.org/doc/html/v4.16/admin-guide/LSM/index.html) (LSMs) are enumerated and each of them must approve the access. Unforetunately if an LSM denies the access there is no uniform Linux reporting mechanism to know which one is responsible. Instead you need to determine which ones are enabled on your system and then investigate each individually. You can determine which LSMs are active by running: `cat /sys/kernel/security/lsm`. Although any LSM could be responsible, [Yama](https://www.kernel.org/doc/html/v4.16/admin-guide/LSM/Yama.html), [SELinux](https://selinuxproject.org/page/Main_Page), and [AppArmor](https://gitlab.com/apparmor/apparmor/-/wikis/GettingStarted) are frequently the relevant ones.
+
+          AppArmor and SELinux both have a rich configuration and reporting mechanisms so its best to view each project's own documentation if you need to learn how to work with them. Yama only has a single configuration setting that can be displayed by running:
+
+          ```
+          cat /proc/sys/kernel/yama/ptrace_scope
+          ```
+
+          This will output a number indicating the current Yama ptrace security policy:
+
+          - 0: Classic ptrace permissions.
+          - 1: Restricted ptrace.
+          - 2: Admin-only attach.
+          - 3: No attach.
+
+          Yama should grant access for createdump under policies 0 and 1, but expect access will be denied under policies 2 and 3. Policy 3 always denies access and policy 2 doesn't work by default because createdump normally does not have the capability CAP_SYS_PTRACE.
```

Comment:
```suggestion
          Yama should grant access for createdump under policies 0 and 1, but expect access will be denied under policies 2 and 3. Policy 3 always denies access, and policy 2 doesn't work by default because createdump normally does not have the capability CAP_SYS_PTRACE.
```